central/pruning/pruning.go

```diff
@@ -365,6 +369,10 @@ func clusterIDsToNegationQuery(clusterIDSet set.FrozenStringSet) *v1.Query {
 }
 
 func (g *garbageCollectorImpl) removeOrphanedProcesses() {
+	g.plops.PruneOrphanedPLOPsByProcessIndicators(pruningCtx, orphanWindow)
```

Comment:
I think the order here is correct. `PruneOrphanedPLOPsByProcessIndicators` needs to come before `PruneOrphanedProcessIndicators`. Otherwise `PruneOrphanedPLOPsByProcessIndicators` wouldn't remove anything, because there would not be any process indicators without pods that would be used to remove PLOPs.

Also `removeOrphanedPLOPs` comes after `removeOrphanedProcesses` so that more PLOPs without a matching process indicator and no process information are removed.